addons/mrp_account/models/stock_move.py

```diff
@@ -47,14 +47,18 @@ def _is_production_consumed(self):
         self.ensure_one()
         return self.location_dest_id.usage == 'production' and self.location_id._should_be_valued()
 
+    def get_svl_without_landed_cost(self):
+        return self.stock_valuation_layer_ids
+
     def _create_out_svl(self, forced_quantity=None):
         svls = super()._create_out_svl(forced_quantity)
         unbuild_svls = svls.filtered('stock_move_id.unbuild_id')
         unbuild_cost_correction_move_list = list()
         for svl in unbuild_svls:
-            build_time_unit_cost = svl.stock_move_id.unbuild_id.mo_id.move_finished_ids.filtered(
+            build_time_svl = svl.stock_move_id.unbuild_id.mo_id.move_finished_ids.filtered(
                 lambda m: m.product_id == svl.product_id
-            ).stock_valuation_layer_ids.unit_cost
+            ).get_svl_without_landed_cost()
```

Comment:
We often filter out layers that impact any other one, it's more generic

By the way, I'm also wondering why we are using such a path (unbuild -> mo -> all finished moves) instead of simply looking at the returned SM, since we are dealing with "unbuilding layers" 
(@Whenrow Hello! How are you ? What do you think ? Quite cold this early September, isn't it ?)

It could then give something like
```py
svl.stock_move_id.origin_returned_move_id.stock_valuation_layer_ids.filtered(lambda l: not l.stock_valuation_layer_id)
```

Reply:
why not. to me it's the same

Reply:
I guess you're already impacted by the sad weather
I will bring you a cookie tomorrow